store/tikv/lock_resolver.go

```diff
@@ -277,23 +277,26 @@ func (lr *LockResolver) BatchResolveLocks(bo *Backoffer, locks []*Lock, loc Regi
 //    commit status.
 // 3) Send `ResolveLock` cmd to the lock's region to resolve all locks belong to
 //    the same transaction.
-func (lr *LockResolver) ResolveLocks(bo *Backoffer, callerStartTS uint64, locks []*Lock) (int64, error) {
+func (lr *LockResolver) ResolveLocks(bo *Backoffer, callerStartTS uint64, locks []*Lock, pushed []uint64) (int64, []uint64, error) {
```

Comment:
I think we better remove the input parameter `pushed`, to make the API easier to use.
ResolveLocks is a very infrequent operation, the cost of allocating a small piece of memory is relatively trial.